### PR TITLE
Remix icons

### PR DESCRIPTION
**Pull request by cristineguadelupe**

### CODE REVIEW

File: lib/assets/data_transform_cell/main.js [hunk 1]
```diff
@@ -8,6 +8,9 @@ export async function init(ctx, payload) {
   ctx.importCSS(
     "https://fonts.googleapis.com/css2?family=Inter:wght@400;500&display=swap"
   );
+  ctx.importCSS(
+    "https://cdn.jsdelivr.net/npm/remixicon@3.2.0/fonts/remixicon.min.css"
```
Comment (josevalim) [inline lib/assets/data_transform_cell/main.js:12]: Shouldn't we keep on using the SVG ones instead of loading all of remix icons? 🤔

Comment (cristineguadelupe) [reply lib/assets/data_transform_cell/main.js:12]: It’s your call!

Comment (josevalim) [reply lib/assets/data_transform_cell/main.js:12]: Any preferences @jonatanklosko?

Comment (josevalim) [reply lib/assets/data_transform_cell/main.js:12]: @cristineguadelupe told me we use the remix icons everywhere else, so let's merge it.

**Review by josevalim — APPROVED** [review]

**Pull request merged**